lib/generators/react_on_rails/bootstrap_generator.rb

```diff
@@ -62,7 +62,7 @@ def strip_application_scss_of_incompatible_sprockets_statements
       end
 
       def add_bootstrap_sprockets_to_gemfile
-        append_to_file("Gemfile", "gem 'bootstrap-sass'\n")
+        append_to_file("Gemfile", "\ngem 'bootstrap-sass'\n")
```

Comment:
Following the same convention we used here: https://github.com/shakacode/react_on_rails/blob/master/lib/generators/react_on_rails/base_generator.rb#L137